documentation/reference.md

```diff
@@ -263,7 +263,7 @@ If you have additional data to report that does not fit any of the columns provi
 |Funding Type:Title|The title of this classification.|string|False|
 |Funding Type:Description|A description of this classification.|string|False|
 |Funding Type:URL|A web link to more details of this classification.|uri|False|
-|Funding Type:Last modified|-|date-time|False|
+|Funding Type:Last modified|The date when information on this classification was last updated|date-time|False|
```

Comment:
This definition is drawn from the 'classification' part of the schema, so it's not easy to make this say 'the date when the information on this funding type was ..."

Will a more generic term do?